src/main/kotlin/org/jetbrains/intellij/platform/gradle/models/ProductInfo.kt

```diff
@@ -175,13 +175,17 @@ internal fun ProductInfo.launchFor(architecture: String): ProductInfo.Launch {
  */
 internal fun String.resolveIdeHomeVariable(platformPath: Path) =
     platformPath.pathString.let {
-        this
-            .replace("\$APP_PACKAGE", it)
+        this.replace("\$APP_PACKAGE", it)
             .replace("\$IDE_HOME", it)
+            .replace("\$IDE_CACHE_DIR", it)
             .replace("%IDE_HOME%", it)
             .replace("Contents/Contents", "Contents")
             .let { entry ->
-                val (_, value) = entry.split("=")
+                val delimiters = "="
+                if (!entry.contains(delimiters)) {
+                    return entry
+                }
+                val (_, value) = entry.split(delimiters)
```

Comment:
Good catch, thank you. I actually often avoid one-liners because later that code is not debug-friendly, even with all assistance the IDE provides. Feel free to submit a change, if you want.